test/test.test.js

```diff
@@ -1126,6 +817,316 @@ test('sign and verify with RSA/ECDSA certificates and global options', function
         })
     })
   })
+  // TODO: Uncomment when (if) fast-jwt adds suport for encoded private keys (passphrase protected)
```

Comment:
Now that nearform/fast-jwt#117 has been merged we can uncomment this.

Reply:
Done